cmd/transferer/main.go

```diff
@@ -20,6 +20,7 @@ func main() {
 
 		clickHouseURL      = cli.Flag("click_house_url", "ClickHouse connection string").String()
 		victoriaMetricsURL = cli.Flag("victoria_metrics_url", "VictoriaMetrics connection string").String()
+		prometheusURL      = cli.Flag("prometheus_url", "Prometheus connection string").String()
```

Comment:
rename it to something like load checker URL

Reply:
resloved in 44c38c52f5f762ad3262986f5032da015d65fb52